app/routes/task_routes.py

```diff
@@ -1 +1,98 @@
-from flask import Blueprint
\ No newline at end of file
+from flask import Blueprint, request, Response, abort, make_response
+import requests
+import os
+from .route_utilities import validate_model, create_model
+from datetime import datetime
+from ..models.task import Task
+from ..db import db
+
+bp=Blueprint("tasks_bp", __name__, url_prefix="/tasks")
+
+@bp.post("")
+def create_task():
+    request_body = request.get_json()
+    response = create_model(Task, request_body)
+    return {"task": response[0]}, response[1]
+
+@bp.get("")
+def get_tasks():
+    query = db.select(Task)
+
+    sort_param = request.args.get("sort")
+
+    if sort_param == "asc":
+        tasks = db.session.scalars(query.order_by(Task.title.asc()))
+    elif sort_param == "desc":
+        tasks = db.session.scalars(query.order_by(Task.title.desc()))
+    else:
+        tasks = db.session.scalars(query.order_by(Task.id))
+
+    tasks_response = []
+    for task in tasks:
+        valid_task = task.to_dict()
+        if not has_special_chars(valid_task["title"]):
+            tasks_response.append(valid_task)
+    return tasks_response
+
+def has_special_chars(title):
+    special_chars = "#$%()*+,-./:;<=>?@[\]^_`{|}~"
+    for char in title:   
+        if char in special_chars:
+            raise ValueError
+    return False
```

Comment:
Why do we need this logic?

How would a request with a request body that has key "title" save a record to the DB in the first place?

Your post request uses `create_model`, which will call a model's `from_dict` method. The `from_dict` method will throw an exception if the request body's keys do not match the keys in the function:
```py
        return cls(
            title=task_data["title"],
            description=task_data["description"],
            completed_at=task_data.get("completed_at")
        )
```

If we can't ever create and save a record with special characters, I think we could skip this logic.